pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -940,33 +940,33 @@ type CloudSpec struct {
 	ProviderName string `json:"providerName"`
 
 	Fake *FakeCloudSpec `json:"fake,omitempty"`
-	// Digitalocean specifies access data to DigitalOcean.
+	// Digitalocean defines thes configuration data of DigitalOcean Cloud provider.
```

Comment:
`thes` should be `the`. Also `of DigitalOcean Cloud provider` should probably be `of the DigitalOcean cloud provider`

Reply:
taken care for all!